homeassistant/components/tuya/strings.json

```diff
@@ -187,6 +187,21 @@
           "4": "Mood 4",
           "5": "Mood 5"
         }
+      },
+      "weather_delay": {
+        "state": {
+          "cancel": "Cancel",
+          "24h": "24h",
+          "48h": "48h",
+          "72h": "72h"
```

Comment:
Actually this value can be from 1 day to maximum 7 days and its always expressed in hours